widgetastic_manageiq/__init__.py

```diff
@@ -5322,33 +5322,46 @@ def is_displayed(self):
         return self.browser.is_displayed(self)
 
 
-class ConversionHostProgress(Widget):
+class ConversionHost(Widget):
     """Represents the progress widget for conversion Host configuration"""
 
     ROOT = './/div[contains(@id,"conversion_hosts")]'
 
     ITEM_LOCATOR = './/div[contains(@class,"list-view-pf-main-info")]'
     ERROR_LOCATOR = './/div/span[contains(@class,"pficon-error-circle-o")]'
     OK_LOCATOR = './/div/span[contains(@class,"pficon-ok")]'
-
+    REMOVE_BUTTON = Button("Remove")
+    DELETE_BUTTON_LOCATOR = './/button[contains(@class,"btn btn-primary")and text()="Remove"]'
```

Comment:
optional:
I would use `Text` like widgets having ClickableMixin. As it use in this widget for clicking on element.